### PR TITLE
Accept regular arguments

### PR DESCRIPTION
**Pull request by levithomason**

### CODE REVIEW

**Comment by jhabdas** [comment]
In review

**Comment by jhabdas** [comment]
Astute back compat. :shitit:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1 +0,0 @@
-{"name":"ChangeLogError","message":"Could not generate changelog for /TechnologyAdvice/stardust. Ensure the repo exists. If it's private, please use a valid access token.","debug":{"cause":{"killed":false,"code":1,"signal":null,"cmd":"/bin/sh -c github_changelog_generator TechnologyAdvice/stardust -t 123"},"isOperational":true,"killed":false,"code":1,"signal":null,"cmd":"/bin/sh -c github_changelog_generator TechnologyAdvice/stardust -t 123"}}
```
Comment (levithomason) [inline CHANGELOG.md]: Previous accidental commit of diff repo changelog during testing :/

**Comment by jhabdas** [comment]
oops ^^

File: README.md [hunk 2]
```diff
@@ -13,10 +13,9 @@ This means you don't need to worry about which shell to use, nor file extensions
 
 ## Usage
 
-    $ ta-script <path_to_script> -- [...script_args]
+    $ ta-script <path_to_script> [...script_args]
```
Comment (levithomason) [inline README.md]: Used to require `--` to separate cli args from the args that belong to the script being called.  This is not intuitive and was forgotten multiple times in real use.  Used a more generic approach now that does not require `--`.

File: cli.js [hunk 3]
```diff
@@ -52,8 +52,12 @@ const throwSearchFail = (script, triedPaths) => {
...
 // ----------------------------------------
 
-const scriptPath = argv._.shift()
-const scriptArgs = argv._.join(' ')
+const scriptPath = process.argv[2]
+const scriptArgs = process.argv.slice(3)
+  // for backward compatibility remove '--' first arg
+  // it was used as the first arg to separate script args from cli args
+  .filter((arg, i) => !(i === 0 && arg === '--'))
+  .join(' ')
```
Comment (levithomason) [inline cli.js]: Here, we are using vanilla `process.argv` for getting our args.  We also trim the starting `--` if present, so old usage still works.

I left in use of `yargs` up top for the nice usage help and required args message.

**Pull request merged**